File: eventual-consistency-support-core/src/main/java/org/xyattic/eventual/consistency/support/core/persistence/ConsumerPersistence.java
package org.xyattic.eventual.consistency.support.core.persistence;

import org.xyattic.eventual.consistency.support.core.consumer.ConsumedMessage;

/**
 * @author wangxing
 * @create 2020/4/14
 */
public interface ConsumerPersistence {

    void save(ConsumedMessage consumedMessage);

}

File: eventual-consistency-support-core/src/main/java/org/xyattic/eventual/consistency/support/core/persistence/ConsumerPersistence.kt
package org.xyattic.eventual.consistency.support.core.persistence

import org.xyattic.eventual.consistency.support.core.consumer.ConsumedMessage

/**
 * @author wangxing
 * @create 2020/4/14
 */
interface ConsumerPersistence {
    fun save(consumedMessage: ConsumedMessage)
}